paths_cli/compiling/strategies.py

```diff
@@ -64,6 +71,8 @@ def _group_parameter(group_name):
         REPLACE_TRUE_PARAMETER
     ],
     name='nearest-neighbor-repex',
+    description=("Use replica exchange only between neearest-neighbor "
+                 "interfaces in this move scheme"),
```

Comment:
typo and missing period
```suggestion
    description=("Use replica exchange only between nearest-neighbor "
                 "interfaces in this move scheme."),
```